schema_utils.py

```diff
@@ -229,47 +261,113 @@ def get(cls, validator_id):
     def has_length_at_least(obj, min_value):
         """Returns True iff the given object (a list) has at least
         `min_value` elements.
+
+        Args:
+            obj: list(str). An object.
```

Comment:
Here and below: prefer "A list of strings" or "A string" to "An object", since the latter conveys no info beyond the name (and actually makes it sound like a dict/class).

Reply:
Done